<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblUser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>USERS :</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableData">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>341</width>
      <height>192</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>1</number>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>150</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>28</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>30</number>
    </attribute>
    <row/>
    <column>
     <property name="text">
      <string>name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>mail</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="updateAccountBtn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>480</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Modifier utilisateur</string>
    </property>
   </widget>
   <widget class="QLabel" name="updateAccountTitle">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mail de l'utilisateur Ã  modifier:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="updateAccountTxt">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="errorZone">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>371</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
